PlayerToken.java

```diff
@@ -0,0 +1,50 @@
+package game.logic;
+enum PlayerToken { NOPLAYER, PLAYER1, PLAYER2 };
+
+
+private class Player {
+	
+	private PlayerToken playerTokens;
+	
+	
+	private int numPieces;
+	
+	private int numPiecesOnBoard;
+	
+	private int numPiecesLeft;
+	
+	
+	
+	
+	public PlayerToken getPlayerToken() {
+		return playerTokens;
+	}
+	
+	
+	public int getPiecesTotal() {
+		return numPiecesTotal;
+	}
+	
+	
+	public int getPiecesOnBoard() {
+		return numPiecesOnBoard;
+	}
+	
+	
+	public int getPiecesLeft() {
+		return numPiecesLeft;
+	}
+	
+	
+	public void placePiece() {
```

Comment:
Maybe this isn't the best name for this function. This name might lead a developer to think that calling this method places a piece on the board, when that is not what the function does. Maybe something along the lines of 'decrementPiecesLeftToPlace' would be better?